### ISSUE TITLE
Page header - hidden elements are visible to screen readers

### ISSUE THREAD

**Issue by jgiardino** [patternfly/patternfly-next#1538]
Currently the elements in the page header become visible/hidden at different screen sizes. It looks like the purpose is to take many separate items that display at larger sizes and collapse them down into a single actions menu. 

The issue is that some of the elements that are hidden are hidden in a way that makes them available to screen readers (e.g. using `.pf-screen-reader`). But this method should only be used when providing an alternate form of information to screen readers when the visible form is not accessible to them (e.g. text alternate for icons in the alert). In this case, the kebab is accessible, therefore an alternate set of actions that are hidden on the screen but accessible to screen readers is unnecessary duplication.

**Comment by rachael-phillips** [comment]
Is this one ready to be closed?

**Comment by mcoker** [comment]
@rachael-phillips nope, the work hasn't been completed. If you're referring to #1521 being merged, that's something different.
